docs/docs/how-tos/nebari-environment-management.md

```diff
@@ -0,0 +1,26 @@
+---
+id: nebari-environment-management
+title: Nebari Environment Management
+description: Best Practices for Managing Your Python Environment
+---
+
+# Nebari Python Environment Management
+
+Nebari configuration and deployment is highly dependent on the version of the Nebari CLI that you have active when you run `nebari` commands.  A such, we highly recommend [installing Conda](https://docs.anaconda.com/free/anaconda/install/) for managing isolated Python environments, especially when working on more than one Nebari deployment from the same machine or using [Nebari extensions][nebari-extension-system].
+
+Once installed, you can use [manage environments](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#managing-environments) in different ways.
+
+This is a simple way to set up a Python environment with a specific Nebari version and exactly one extension installed.  *When **pip** is installed with conda, the packages will not be saved in the global namespace*.  This especially critical when working with extensions, because `nebari` will detect and apply any extensions installed in your Python environment when it runs.
+
```

Comment:
I will add a new line here, saying that we will create an environment called `nebari0`, which will "store" all of our packages for those not used to conda.

Reply:
I'm not sure what this is referring to.  I've never seen `nebari0` when I run  `conda env list` in any of my locals where I've been deploying Nebari.

What is nebari0 and how does it relate?

Reply:
I think "nebari0" is an example name for a proposed conda environment used to namespace the package dependencies needed to install Nebari.

Reply:
Yeah, it could be anything, for example `nebari-dev' would be used to install nebari in a experimental fashion. I assume they added the extra `0` in the example `nebari0` because the text above discusses using multiple environments with possible different package versions.

Reply:
Oh my bad, I forgot that the example used `nebari0` as the conda name and that totally threw me off parsing the comment.  My second commit will include the language tweaks